charts/dapr/charts/dapr_scheduler/templates/dapr_scheduler_poddisruptionbudget.yaml

```
@@ -0,0 +1,32 @@
+{{- if and (gt (int .Values.replicaCount) 1) (eq .Values.global.scheduler.enabled true) }}
+{{- if .Capabilities.APIVersions.Has "policy/v1" }}
+apiVersion: policy/v1
+{{- else }}
+apiVersion: policy/v1beta1
+{{- end }}
+kind: PodDisruptionBudget
+metadata:
+  name: dapr-scheduler-server-disruption-budget
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: dapr-scheduler-server
+    {{- range $key, $value := .Values.global.k8sLabels }}
+    {{ $key }}: {{ tpl $value $ }}
+    {{- end }}
+spec:
+{{- if .Values.global.ha.disruption.minimumAvailable }}
+  minAvailable: {{ .Values.global.ha.disruption.minimumAvailable }}
+{{- end }}
+{{- if .Values.global.ha.disruption.maximumUnavailable }}
+  maxUnavailable: {{ .Values.global.ha.disruption.maximumUnavailable }}
+{{- end }}
+  selector:
+    matchLabels:
+      app: dapr-scheduler-server
+      {{- range $key, $value := .Values.global.k8sLabels }}
+      {{ $key }}: {{ tpl $value $ }}
+      {{- end }}
+      {{- with .Values.global.labels }}
+        {{- toYaml . | nindent 6 }}
```

Shouldn't this be 8?

---

based on sentry, looks like 6 is fine: https://github.com/dapr/dapr/blob/master/charts/dapr/charts/dapr_sentry/templates/dapr_sentry_poddisruptionbudget.yaml#L30

---

Ok I misunderstood the code. The issue is that there are 8 spaces at the start of the line and it caused confusion.
```suggestion
      {{- toYaml . | nindent 6 }}
```

---

It works as is, are you saying there is an issue with indentation? It is consistent with the other pod disruption budgets in dapr.

---

It works because there's the `-` at the start of `{{`, but it is _technically_ confusing (since the line has 8 spaces at the start instead of 6). So more like a nit now.

---

I appreciate the nit. For consistency sake, I will leave this as is because all the other pod disruption budgets in dapr (specifically the `operator`, `injector`, `sentry`, and `placement`) all have this exact config. To avoid confusion, maybe all of them should be updated with your suggestion. However, I would like to avoid updating all of them in the scheduler PR